eng/Version.Details.xml

```diff
@@ -150,5 +150,29 @@
       <Uri>https://github.com/dotnet/runtime-assets</Uri>
       <Sha>4e8d82a2bdc31db2a3d392b4021b0c9ab19a4228</Sha>
     </Dependency>
+    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Sdk" Version="6.0.1-alpha.1.20073.2">
+      <Uri>https://github.com/dotnet/llvm-project</Uri>
+      <Sha>320e484510d00c6112cfe972e3758fdcaa955864</Sha>
+    </Dependency>
+    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Tools" Version="6.0.1-alpha.1.20073.2">
+      <Uri>https://github.com/dotnet/llvm-project</Uri>
+      <Sha>320e484510d00c6112cfe972e3758fdcaa955864</Sha>
+    </Dependency>
+    <Dependency Name="runtime.win-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Sdk" Version="6.0.1-alpha.1.20073.2">
+      <Uri>https://github.com/dotnet/llvm-project</Uri>
+      <Sha>320e484510d00c6112cfe972e3758fdcaa955864</Sha>
+    </Dependency>
+    <Dependency Name="runtime.win-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Tools" Version="6.0.1-alpha.1.20073.2">
+      <Uri>https://github.com/dotnet/llvm-project</Uri>
+      <Sha>320e484510d00c6112cfe972e3758fdcaa955864</Sha>
+    </Dependency>
+    <Dependency Name="runtime.osx.10.12-x64.Microsoft.NETCore.Runtime.Mono.LLVM.Sdk" Version="6.0.1-alpha.1.20073.2">
```

Comment:
Why does osx have to have the version in?

Reply:
I have no idea. I copy-pasted that logic from Mono into LLVM: https://github.com/dotnet/runtime/blob/master/src/mono/netcore/nuget/Directory.Build.props#L17 -> https://github.com/dotnet/llvm-project/blob/release/6.x/nuget/Directory.Build.props#L14